ovs/lib/albamigration.py

```diff
@@ -277,14 +280,12 @@ def migrate():
         # Regenerate maintenance service
         try:
             for node in AlbaNodeList.get_albanodes():
-                _local_client = SSHClient(node.ip, node.username)
-                _service_manager = ServiceFactory.get_manager()
-
-                for backen_name, maintenance_services in node.maintenance_services.iteritems():
-                    for maintenance_name in maintenance_services:
-                        _service_manager.regenerate_service(name='alba-maintenance',
-                                                            client=_local_client,
-                                                            target_name=maintenance_name)
+                for backend_name, maintenance_services in node.maintenance_services.iteritems():
```

Comment:
Maintenance services is a dynamic property which uses the API of the ASD-manager.
Config management can be used to even change it for ASD-managers that are down :o

https://imgflip.com/i/2u5ks8